### ISSUE TITLE
CARE example has issues in training

### ISSUE THREAD

**Issue by jdeschamps** [CAREamics/careamics-napari#1]
On linux virtual machine, training returns right away. Clicking on training again leads to two training in parallel.

**Comment by mese79** [comment]
Hmm, I couldn't reproduce it!  
Maybe it had something to do with the `CAREamics` itself...

**Comment by jdeschamps** [comment]
Yes, could be!

Could you confirm that if you start training CARE, the "train" button changes to "stop"? And you can stop it without problem and restart training without apparent issues?

**Comment by mese79** [comment]
Yeah, it seems to be working with many stop/restart/reset that I did!

**Comment by jdeschamps** [comment]
Great! Then we close this!
